src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -1079,6 +1079,31 @@ Transactor::checkMultiSign(PreclaimContext const& ctx)
 
 //------------------------------------------------------------------------------
 
+// increment the touch counter on an account
+static void
+touchAccount(
+    ApplyView& view,
+    AccountID const& id,
+    std::optional<std::shared_ptr<SLE>> sle)
```

Comment:
What is the intent of this argument that is not actually used?

Reply:
Which argument?

Reply:
@RichardAH I removed this. Can you re-review? Was this done so that later you could call it from another location and include the sle? Can we just remove that and add it later if we need to?

Reply:
It was forward thinking for other touches where the SLE is already provided by the calling function.
Speaking of which, I just did a quick code review and don't we need touch in a few other places not just unfunded offers?